README.md

```diff
@@ -81,6 +69,19 @@ import { makeRollupConfig } from "@jspsych/config/rollup";
 export default makeRollupConfig("<camelcase name of new package>");
 ```
 
+Edit `rollup.config.dev.mjs`:
+
+```mjs
+import { makeRollupConfig } from "@jspsych/config/rollup";
+
+import { makeDevConfig } from "../../rollup-dev.mjs";
+
+const rollupConfig = makeRollupConfig("lookitInitJsPsych");
```

Comment:
If my assumption is correct that the package name passed to `makeRollupConfig` needs to change for each package, then I'd like to note that here too.

Reply:
No problem.  This has been added.